services/checks/checks.go

```diff
@@ -346,6 +363,9 @@ func (s *Service) executeMySQLChecks(ctx context.Context) ([]string, error) {
 				s.l.Warnf("Failed to process action result: %s", err)
 				continue
 			}
+			// record metrics
+			s.mExecutedScripts.WithLabelValues(string(models.MySQLServiceType)).Inc()
+			s.mGeneratedAlerts.WithLabelValues(string(models.MySQLServiceType), string(c.Type)).Inc()
```

Comment:
One check can produce multiple alerts, moreover `alerts` can be empty at all. So I think that counter should be incremented on `len(alerts)`.